tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
-envlist = py27,py34,py35,py36
+envlist = py34,py35,py36,py38
```

Comment:
I think we can remove `py34` here too, since we dropped support for Python 3.4 in an earlier commit but forgot to update the `tox.ini`.